core/src/sql/access_type.rs

```diff
@@ -80,7 +122,45 @@ impl Default for JwtAccess {
 	}
 }
 
+impl Display for JwtAccess {
+	fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+		match &self.verify {
+			JwtAccessVerify::Key(ref v) => {
+				write!(f, "ALGORITHM {} KEY {}", v.alg, quote_str(&v.key))?;
+			}
+			JwtAccessVerify::Jwks(ref v) => {
+				write!(f, "JWKS {}", quote_str(&v.url),)?;
+			}
+		}
+		if let Some(iss) = &self.issue {
+			write!(f, " WITH ISSUER KEY {}", quote_str(&iss.key))?;
+		}
+		Ok(())
+	}
+}
+
+impl InfoStructure for JwtAccess {
+	fn structure(self) -> Value {
+		Value::from(map! {
+			"verify".to_string() => match self.verify {
+				JwtAccessVerify::Jwks(v) => Value::from(map!{
+					"jwks".to_string() => v.url.into(),
+				}),
+				JwtAccessVerify::Key(v) => Value::from(map!{
+					"alg".to_string() => v.alg.structure(),
+					"key".to_string() => v.key.into(),
+				}),
+			},
+			"issuer".to_string(), if let Some(v) = self.issue => Value::from(map!{
+				"alg".to_string() => v.alg.structure(),
+				"key".to_string() => v.key.into(),
+			}),
+		})
+	}
+}
+
 impl JwtAccess {
+	/// Redacts certain parts of the definition for security on export.
```

Comment:
Thank you for adding this comment. Just a nitpick, but this is not called on export, only on `INFO`.

Reply:
I'll change in an upcoming PR!